compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/NodeFinder.java

```diff
@@ -302,12 +302,6 @@ public void visit(BLangXMLNS xmlnsNode) {
     @Override
     public void visit(BLangFunction funcNode) {
         lookupNodes(funcNode.annAttachments);
-        // Compare the target lookup pos with the function symbol pos to ensure that we are not looking for the
```

Comment:
Why do we need this change?

Reply:
It's added due to a pull from master. Took a pull from the `java_17_migration` branch.